ssvsigner/web3signer/web3signer.go

```diff
@@ -0,0 +1,95 @@
+package web3signer
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/carlmjohnson/requests"
+)
+
+type Web3Signer struct {
+	baseURL    string
+	httpClient *http.Client
+}
+
+func New(baseURL string) *Web3Signer {
+	baseURL = strings.TrimRight(baseURL, "/")
+
+	return &Web3Signer{
+		baseURL: baseURL,
+		httpClient: &http.Client{
+			Timeout: 30 * time.Second,
```

Comment:
@nkryuchkov is this maybe too high? given we now lock for signing

unless we pass timeoutted context ofc instead of relying on this